src/main/java/uk/gov/companieshouse/filinghistory/consumer/apiclient/FilingHistoryApiClient.java

```diff
@@ -41,15 +48,23 @@ public void upsertFilingHistory(InternalFilingHistoryApi requestBody) {
         }
     }
 
-    public void deleteFilingHistory(String entityId) {
+    public void deleteFilingHistory(DeleteApiClientRequest apiClientRequest) {
+        String deltaAt = apiClientRequest.deltaAt();
+        if (StringUtils.isBlank(deltaAt)) {
```

Comment:
Validation here is a bit overkill, chs-delta-api already does extensive validation. If necessary it could be separated into a new validator class which does validation for the whole delta model.

Reply:
Resolved in latest commit